src/BuildScriptGenerator/Options/BuildScriptGeneratorOptions.cs

```diff
@@ -82,5 +82,7 @@ public class BuildScriptGeneratorOptions
         public string CustomRequirementsTxtPath { get; set; }
 
         public string DebianFlavor { get; set; }
+
+        public string UserCallerId { get; set; }
```

Comment:
From our discussion, I thought that we would go with "CallerId", as anything "User" related looks like it actually corresponds to the end-users, our customers, and not the partner team relying on Oryx.
Did you think about another scenario where having User in the name would be valuable?

Reply:
Dan, during our discussion I thought by `CallerId` you tried to explain the purpose of this environment variable. Did not realize I should change the name of the variable.
Getting rid of the `user` prefix from the variable name makes more sense to me as well. I will make this change.

Reply:
Thanks!